ajax/libs/slim.js/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "slim.js",
+  "filename": "slim.js",
```

Comment:
`"filename"` should point to the main minified file. It will be `"Slim.min.js"`.